warehouse/views.py

```diff
@@ -405,8 +405,9 @@ def stats(request):
     )
     # Move top packages into a dict to make JSON more self describing
     top_packages = {
-        pkg_name: {"size": int(pkg_bytes) if pkg_bytes is not None else 0}
+        pkg_name: {"size": int(pkg_bytes)}
         for pkg_name, pkg_bytes in top_100_packages
+        if pkg_bytes not in [0, None]
```

Comment:
It seems like this would result in less than 100 packages being displayed. Maybe the query should be adjusted instead?

Reply:
Yep, thanks. I've modified the query to filter for above 0 packages.